godtools/App/Features/Tools/ToolsMenuView.swift

```diff
@@ -53,6 +53,8 @@ class ToolsMenuView: UIViewController {
             target: self,
             action: #selector(handleMenu(barButtonItem:))
         )
+        
+        viewModel.openSetupParallelLanguageifAvailable()
```

Comment:
Hey @reldredge71 I don't think we want to present the setup parallel language anytime this view loads.  I noticed each time I relaunch the app the setup parallel language is presented over the tools.
Can this be part of the OnboardingFlow.  Similar to how quick actions is part of the onboarding flow.  It would look smoother if this were pushed onto the onboarding flow navigation stack when leaving quick actions.